keystone-page.c

```diff
@@ -40,15 +40,18 @@ int epm_init(struct epm* epm, unsigned int min_pages)
   count = 0x1 << order;
 
   /* prevent kernel from complaining about an invalid argument */
-  if (order <= MAX_ORDER)
-    epm_vaddr = (vaddr_t) __get_free_pages(GFP_HIGHUSER, order);
-
+  if (order <= MAX_ORDER) {
```

Comment:
I guess this is temporary, so please add a comment that indicates it

Reply:
Added comments